File: 242.py
# s = "ab"
# t = "a"
# if len(s) == len(t):
#     for i in range(len(t)):
#         s = s.replace(t[i], '', 1)
#     print(len(s))
# else:
#     print(-1)

class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        if len(s) == len(t):
            for i in range(len(t)):
                s = s.replace(t[i], '', 1)
            return len(s) == 0
        return False

File: 242.cpp
#include <string>
using namespace std;

class Solution {
public:
    bool isAnagram(string s, string t) {
        int count[26] = {0};
        for (char c : s) {
            count[c - 'a']++; 
        }

        for (char c : t) {
            count[c - 'a']--; 
        }

        for (int i : count) {
            if (i != 0) {
                return false;
            }
        }
        return true;
    }
};